package-lock.json

```diff
@@ -48,7 +48,6 @@
         "panzoom": "^9.4.3",
         "rc-slider": "^10.1.0",
         "react": "^18.2.0",
-        "react-bs-notifier": "^7.0.0",
```

Comment:
I was surprised of seeing only one line change in this file. FYI it's not supposed to be edited by hand.
You need to run `npm rm --save react-bs-notifier` in order to  let NPM modify the file itself and remove unneeded sub-dependencies.

I took the liberty of pushing my local changes after doing that myself to make sure I wasn't missing something :p

Reply:
Sorry for not doing that. Would keep in mind.